.github/workflows/docker-push.yml

```
@@ -0,0 +1,101 @@
+    needs: [check_secrets]
+    permissions:
+      contents: read
+      packages: write
+    if: needs.check_secrets.outputs.docker == 'true'
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+      with:
+        fetch-depth: 0
+        ref: ${{ inputs.ref }}
+    - uses: ./.github/actions/install-nix-action
```

actionlint (1.7.10)

missing input "github_token" which is required by action "Install Nix" defined at "./.github/actions/install-nix-action". all required inputs are "dogfood", "github_token"

(action)